packages/core/test/deploymentBuilder/value.ts

```diff
@@ -1,15 +1,20 @@
 /* eslint-disable import/no-unused-modules */
-import { assert } from "chai";
-import { ethers } from "ethers";
-
-import { buildModule } from "dsl/buildModule";
-import { generateDeploymentGraphFrom } from "process/generateDeploymentGraphFrom";
 import type {
   IDeploymentGraph,
   IDeploymentBuilder,
-} from "types/deploymentGraph";
-import type { Artifact } from "types/hardhat";
-import { isArtifactContract, isHardhatContract, isCall } from "utils/guards";
+} from "../../src/types/deploymentGraph";
+import type { Artifact } from "../../src/types/hardhat";
+
+import { assert } from "chai";
+import { ethers } from "ethers";
+
+import { buildModule } from "../../src/dsl/buildModule";
+import { generateDeploymentGraphFrom } from "../../src/process/generateDeploymentGraphFrom";
```

Comment:
Noticed this in many files. Why doesn't the linter auto-orders this? There's a pretty random order here, wheres before external dependencies were first, and then the local ones.

Reply:
I think the rule has changed slightly with a newer version. It appears to sort `import type` before `import` even if that means internal before external, or more correctly I think it is:

* import type external
* import type internal
* import external
* import internal

But I assume we can change the config settings if we want?

Reply:
Ohhh, it must be a versioning issue! I hate when this happens 😫

I hadn't noticed the `type` in the first import. I guess it makes sense now. thanks

Reply:
For future reference, the config is in global eslint config:

![image](https://user-images.githubusercontent.com/24030/225288104-b70fccf8-9f96-4ab0-a33a-c1f1dacb967d.png)

You can see that type imports take priority. We can reconfigure this and `eslint:fix` will take care of the rest.